flutter/lib/src/sentry_flutter.dart

```diff
@@ -43,6 +48,9 @@ mixin SentryFlutter {
   }) async {
     final flutterOptions = SentryFlutterOptions();
 
+    // ignore: invalid_use_of_internal_member
+    mainIsolateStartTime = flutterOptions.clock();
```

Comment:
why are you setting it here?
This changes its value, and it shouldn't, as far as i understand

Reply:
the main problem is if `SentryFlutter.init` is not called immediately. in that case the difference would be higher.
So yes, the static variable is enough